handle_mirror_webhook.py

```diff
@@ -1,32 +1,62 @@
+import git
 import hglib
+import logging
 import os
-from flask import Flask
+from tornado import gen
+from tornado.ioloop import IOLoop
+from tornado.web import RequestHandler, Application
 
-app = Flask(__name__)
-app.config['DEBUG'] = False
-
-def get_environment_variable(variable, default):
-    try:
-        return os.environ[variable]
-    except KeyError:
-        return default
-
-REPO_PATH = get_environment_variable('REPO_PATH', './yt-hg')
-BB_REPO = get_environment_variable('BB_REPO', 'https://bitbucket.org/ngoldbaum/yt')
-GH_REPO = get_environment_variable(
+GH_REPO_PATH = os.environ.get('GH_REPO_PATH', '/tmp/yt-git')
+BB_REPO_PATH = os.environ.get('BB_REPO_PATH', '/tmp/yt-hg')
```

Comment:
naming these two `GH` and `BB` is a bit confusing since they're not really on github and bitbucket, maybe `LOCAL_GIT_REPO_PATH` and `LOCAL_HG_REPO_PATH`?